comp/autoscaling/workload/impl/config_retriever.go

```diff
@@ -5,7 +5,7 @@
 
 //go:build kubeapiserver
 
-package workload
+package impl
```

Comment:
```suggestion
package workloadimpl
```

For all the files in this directory.

Reply:
Done 👍